<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="showText">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Эксперемент с раз</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="inputText">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>рывом текста...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>Вывод информации</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="start">
    <property name="title">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QMenu" name="regime">
    <property name="title">
     <string>Режимы</string>
    </property>
   </widget>
   <widget class="QMenu" name="students">
    <property name="title">
     <string>Ученики</string>
    </property>
   </widget>
   <widget class="QMenu" name="setting">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="start"/>
   <addaction name="regime"/>
   <addaction name="students"/>
   <addaction name="setting"/>
   <addaction name="help"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
